deploy/docker/entrypoint.sh

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env bash
 
+if [ $FILESTORE_IP_ADDRESS ];
+then
+        ## Trim APPSMITH_FILESTORE_IP and FILE_SHARE_NAME
+        FILESTORE_IP_ADDRESS=$(echo "$FILESTORE_IP_ADDRESS" | xargs)
+        FILE_SHARE_NAME=$(echo "$FILE_SHARE_NAME" | xargs)
+
+        echo "Installing appsmith for cloudRun"
```

Comment:
Not really _installing_, right? We're just running, no? 🤔

```suggestion
        echo "Running Appsmith for CloudRun"
```

Reply:
May be deploying appsmith for cloudrun ?